src/sentry/monitors/serializers.py

```diff
@@ -39,13 +38,13 @@ def serialize(self, obj, attrs, user, **kwargs) -> MonitorEnvironmentSerializerR
 
 class MonitorConfigSerializerResponse(TypedDict):
     schedule_type: Literal["crontab", "interval"]
-    schedule: str | Tuple[int, IntervalNames]
```

Comment:
this file is only in this change because this was the first converted annotation which triggered apidocs to fail its build